dist/index.js

```diff
@@ -1026,13 +1026,17 @@ function executeO365CLICommand(command) {
         };
         try {
             yield exec_1.exec(`"${o365CLIPath}" ${command}`, [], options);
-            return o365CLICommandOutput;
+            return cleanOutput ? cleanString(o365CLICommandOutput) : o365CLICommandOutput;
         }
         catch (err) {
             throw new Error(err);
         }
     });
 }
+// remove any line breaks and spaces in a string
+function cleanString(input) {
+    return input.replace(/[\r\n\s]+/gm, "");
```

Comment:
Shall we adjust it to remove only leading/trailing spaces/new lines? If there is a space or a new line inside the string, there is probably a reason for it and removing them could make the output unusable.

Reply:
Sure @waldekmastykarz I'll push the updated code.

Reply:
@waldekmastykarz - The code is now updated with the `trim()` method which satisfies the requirement of removing leading/trailing spaces/new lines from the output.